### ISSUE TITLE
I have a question

### ISSUE THREAD

**Issue by MPLAP** [carpedm20/DCGAN-tensorflow#358]
Training on the dcgan, The learning progress window looks like this.

1Epoch: [ 0/25]  [ 0 / 116] time:       d_loss:           g_loss:

I am curious about this progress window.
Could you please expain more about this?

The parameters are as follows

sample_num = 100 , input_height,width = 256 output_height,width=64 sample_freq=100
ckpt_freq=200 

IThe total number of data is 116.

**Comment by Muhammad057** [comment]
Epoch: [0/25] tells you for how many number of epochs the model shall be trained. You can change it from main.py
[0/116] are your number of training data
d_loss and g_loss are the loss function values for your data
